<ui version="4.0" >
 <class>DlgEditConnection</class>
 <widget class="QDialog" name="DlgEditConnection" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Connection</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <property name="horizontalSpacing" >
      <number>6</number>
     </property>
     <property name="verticalSpacing" >
      <number>6</number>
     </property>
     <item row="2" column="0" >
      <widget class="QLabel" name="Caption_2" >
       <property name="text" >
        <string>Host</string>
       </property>
       <property name="buddy" >
        <cstring>edHost</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="cbxDriver" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="Caption_3" >
       <property name="text" >
        <string>Driver</string>
       </property>
       <property name="buddy" >
        <cstring>cbxDriver</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLineEdit" name="edUser" />
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="edHost" />
     </item>
     <item row="6" column="1" >
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="edDatabase" />
       </item>
       <item>
        <widget class="QPushButton" name="btFindDatabaseFile" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>32</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="Caption_2_2_2" >
       <property name="text" >
        <string>User</string>
       </property>
       <property name="buddy" >
        <cstring>edUser</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="edPort" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="Caption_2_2" >
       <property name="text" >
        <string>Port</string>
       </property>
       <property name="buddy" >
        <cstring>edPort</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="edCaption" />
     </item>
     <item row="6" column="0" >
      <widget class="QLabel" name="Caption_2_2_2_2_2" >
       <property name="text" >
        <string>Database</string>
       </property>
       <property name="buddy" >
        <cstring>edDatabase</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="Caption_2_2_2_2" >
       <property name="text" >
        <string>Password</string>
       </property>
       <property name="buddy" >
        <cstring>edPassword</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QLineEdit" name="edPassword" >
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="Caption" >
       <property name="text" >
        <string>Caption</string>
       </property>
       <property name="buddy" >
        <cstring>edCaption</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Codec</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" >
      <widget class="QComboBox" name="lstCodec" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackOptions" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QVBoxLayout" >
       <property name="leftMargin" >
        <number>2</number>
       </property>
       <property name="topMargin" >
        <number>2</number>
       </property>
       <property name="rightMargin" >
        <number>2</number>
       </property>
       <property name="bottomMargin" >
        <number>2</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>MySQL options</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Set names</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="lstMySqlSetNames" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>SQLite options</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QCheckBox" name="sqlite_chkPragma_short_column_names" >
            <property name="text" >
             <string>PRAGMA short_column_names</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="sqlite_chkPragma_full_column_names" >
            <property name="text" >
             <string>PRAGMA full_column_names</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>264</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>1</number>
     </property>
     <property name="leftMargin" >
      <number>1</number>
     </property>
     <property name="topMargin" >
      <number>1</number>
     </property>
     <property name="rightMargin" >
      <number>1</number>
     </property>
     <property name="bottomMargin" >
      <number>1</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btOk" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edCaption</tabstop>
  <tabstop>edHost</tabstop>
  <tabstop>edPort</tabstop>
  <tabstop>edUser</tabstop>
  <tabstop>edPassword</tabstop>
  <tabstop>edDatabase</tabstop>
  <tabstop>btOk</tabstop>
  <tabstop>btCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btOk</sender>
   <signal>clicked()</signal>
   <receiver>DlgEditConnection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>79</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel" >
     <x>54</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgEditConnection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>152</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel" >
     <x>132</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
